docs/guides/qedma-qesem.ipynb

```diff
@@ -37,40 +37,56 @@
    "id": "74823696",
```

Comment:
<p>So the default behavior is resource estimation?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Qiskit/documentation/pull/2139/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
I see this is already commented on above. So, ignore.